app/src/main/res/layout/fragment_message_list.xml

```diff
@@ -4,10 +4,9 @@
     android:layout_height="match_parent"
     tools:context=".activities.MessageListFragment" >
 
-    <ListView
-        android:id="@+id/listview"
+    <android.support.v7.widget.RecyclerView
+        android:id="@+id/recyclerView"
         android:layout_width="match_parent"
-        android:layout_height="fill_parent" >
-    </ListView>
-
+        android:layout_height="fill_parent"
```

Comment:
Committed as is, this changeset will break the app (cause runtime failures).  Wherever possible, it's good practice to keep XML changes in the same commit as the Java code changes that you make as well.